### PR TITLE
Fix githupages

### PR DESCRIPTION
**Pull request by subho007**

### CODE REVIEW

File: _site/js/jquery.fancybox.pack.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/*! fancyBox v2.1.5 fancyapps.com | fancyapps.com/fancybox/#license */
+(function(r,G,f,v){var J=f("html"),n=f(r),p=f(G),b=f.fancybox=function(){b.open.apply(this,arguments)},I=navigator.userAgent.match(/msie/i),B=null,s=G.createTouch!==v,t=function(a){return a&&a.hasOwnProperty&&a instanceof f},q=function(a){return a&&"string"===f.type(a)},E=function(a){return q(a)&&0<a.indexOf("%")},l=function(a,d){var e=parseInt(a,10)||0;d&&E(a)&&(e*=b.getViewport()[d]/100);return Math.ceil(e)},w=function(a,b){return l(a,b)+"px"};f.extend(b,{version:"2.1.5",defaults:{padding:15,margin:20,
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:2]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: _site/js/jquery.fancybox.pack.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+openMethod:"zoomIn",closeEffect:"fade",closeSpeed:250,closeEasing:"swing",closeOpacity:!0,closeMethod:"zoomOut",nextEffect:"elastic",nextSpeed:250,nextEasing:"swing",nextMethod:"changeIn",prevEffect:"elastic",prevSpeed:250,prevEasing:"swing",prevMethod:"changeOut",helpers:{overlay:!0,title:!0},onCancel:f.noop,beforeLoad:f.noop,afterLoad:f.noop,beforeShow:f.noop,afterShow:f.noop,beforeChange:f.noop,beforeClose:f.noop,afterClose:f.noop},group:{},opts:{},previous:null,coming:null,current:null,isActive:!1,
+isOpen:!1,isOpened:!1,wrap:null,skin:null,outer:null,inner:null,player:{timer:null,isActive:!1},ajaxLoad:null,imgPreload:null,transitions:{},helpers:{},open:function(a,d){if(a&&(f.isPlainObject(d)||(d={}),!1!==b.close(!0)))return f.isArray(a)||(a=t(a)?f(a).get():[a]),f.each(a,function(e,c){var k={},g,h,j,m,l;"object"===f.type(c)&&(c.nodeType&&(c=f(c)),t(c)?(k={href:c.data("fancybox-href")||c.attr("href"),title:c.data("fancybox-title")||c.attr("title"),isDom:!0,element:c},f.metadata&&f.extend(!0,k,
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:7]: **[jshint]**
-  Confusing use of '!'.

File: _site/js/jquery.fancybox.pack.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+c.metadata())):k=c);g=d.href||k.href||(q(c)?c:null);h=d.title!==v?d.title:k.title||"";m=(j=d.content||k.content)?"html":d.type||k.type;!m&&k.isDom&&(m=c.data("fancybox-type"),m||(m=(m=c.prop("class").match(/fancybox\.(\w+)/))?m[1]:null));q(g)&&(m||(b.isImage(g)?m="image":b.isSWF(g)?m="swf":"#"===g.charAt(0)?m="inline":q(c)&&(m="html",j=c)),"ajax"===m&&(l=g.split(/\s+/,2),g=l.shift(),l=l.shift()));j||("inline"===m?g?j=f(q(g)?g.replace(/.*(?=#[^\s]+$)/,""):g):k.isDom&&(j=c):"html"===m?j=g:!m&&(!g&&
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:8]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: _site/js/jquery.fancybox.pack.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+k.isDom)&&(m="inline",j=c));f.extend(k,{href:g,type:m,content:j,title:h,selector:l});a[e]=k}),b.opts=f.extend(!0,{},b.defaults,d),d.keys!==v&&(b.opts.keys=d.keys?f.extend({},b.defaults.keys,d.keys):!1),b.group=a,b._start(b.opts.index)},cancel:function(){var a=b.coming;a&&!1!==b.trigger("onCancel")&&(b.hideLoading(),b.ajaxLoad&&b.ajaxLoad.abort(),b.ajaxLoad=null,b.imgPreload&&(b.imgPreload.onload=b.imgPreload.onerror=null),a.wrap&&a.wrap.stop(!0,!0).trigger("onReset").remove(),b.coming=null,b.current||
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:9]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.
-  Confusing use of '!'.

File: _site/js/jquery.fancybox.pack.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+b._afterZoomOut(a))},close:function(a){b.cancel();!1!==b.trigger("beforeClose")&&(b.unbindEvents(),b.isActive&&(!b.isOpen||!0===a?(f(".fancybox-wrap").stop(!0).trigger("onReset").remove(),b._afterZoomOut()):(b.isOpen=b.isOpened=!1,b.isClosing=!0,f(".fancybox-item, .fancybox-nav").remove(),b.wrap.stop(!0,!0).removeClass("fancybox-opened"),b.transitions[b.current.closeMethod]())))},play:function(a){var d=function(){clearTimeout(b.player.timer)},e=function(){d();b.current&&b.player.isActive&&(b.player.timer=
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:10]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.
-  Confusing use of '!'.

File: _site/js/jquery.fancybox.pack.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+setTimeout(b.next,b.current.playSpeed))},c=function(){d();p.unbind(".player");b.player.isActive=!1;b.trigger("onPlayEnd")};if(!0===a||!b.player.isActive&&!1!==a){if(b.current&&(b.current.loop||b.current.index<b.group.length-1))b.player.isActive=!0,p.bind({"onCancel.player beforeClose.player":c,"onUpdate.player":e,"beforeLoad.player":d}),e(),b.trigger("onPlayStart")}else c()},next:function(a){var d=b.current;d&&(q(a)||(a=d.direction.next),b.jumpto(d.index+1,a,"next"))},prev:function(a){var d=b.current;
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:11]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.
-  Confusing use of '!'.

File: _site/js/jquery.fancybox.pack.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+d&&(q(a)||(a=d.direction.prev),b.jumpto(d.index-1,a,"prev"))},jumpto:function(a,d,e){var c=b.current;c&&(a=l(a),b.direction=d||c.direction[a>=c.index?"next":"prev"],b.router=e||"jumpto",c.loop&&(0>a&&(a=c.group.length+a%c.group.length),a%=c.group.length),c.group[a]!==v&&(b.cancel(),b._start(a)))},reposition:function(a,d){var e=b.current,c=e?e.wrap:null,k;c&&(k=b._getPosition(d),a&&"scroll"===a.type?(delete k.position,c.stop(!0,!0).animate(k,200)):(c.css(k),e.pos=f.extend({},e.dim,k)))},update:function(a){var d=
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:12]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: _site/js/jquery.fancybox.pack.js [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+a&&a.type,e=!d||"orientationchange"===d;e&&(clearTimeout(B),B=null);b.isOpen&&!B&&(B=setTimeout(function(){var c=b.current;c&&!b.isClosing&&(b.wrap.removeClass("fancybox-tmp"),(e||"load"===d||"resize"===d&&c.autoResize)&&b._setDimension(),"scroll"===d&&c.canShrink||b.reposition(a),b.trigger("onUpdate"),B=null)},e&&!s?0:300))},toggle:function(a){b.isOpen&&(b.current.fitToView="boolean"===f.type(a)?a:!b.current.fitToView,s&&(b.wrap.removeAttr("style").addClass("fancybox-tmp"),b.trigger("onUpdate")),
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:13]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: _site/js/jquery.fancybox.pack.js [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+b.update())},hideLoading:function(){p.unbind(".loading");f("#fancybox-loading").remove()},showLoading:function(){var a,d;b.hideLoading();a=f('<div id="fancybox-loading"><div></div></div>').click(b.cancel).appendTo("body");p.bind("keydown.loading",function(a){if(27===(a.which||a.keyCode))a.preventDefault(),b.cancel()});b.defaults.fixed||(d=b.getViewport(),a.css({position:"absolute",top:0.5*d.h+d.y,left:0.5*d.w+d.x}))},getViewport:function(){var a=b.current&&b.current.locked||!1,d={x:n.scrollLeft(),
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:14]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: _site/js/jquery.fancybox.pack.js [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+y:n.scrollTop()};a?(d.w=a[0].clientWidth,d.h=a[0].clientHeight):(d.w=s&&r.innerWidth?r.innerWidth:n.width(),d.h=s&&r.innerHeight?r.innerHeight:n.height());return d},unbindEvents:function(){b.wrap&&t(b.wrap)&&b.wrap.unbind(".fb");p.unbind(".fb");n.unbind(".fb")},bindEvents:function(){var a=b.current,d;a&&(n.bind("orientationchange.fb"+(s?"":" resize.fb")+(a.autoCenter&&!a.locked?" scroll.fb":""),b.update),(d=a.keys)&&p.bind("keydown.fb",function(e){var c=e.which||e.keyCode,k=e.target||e.srcElement;
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:15]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: _site/js/jquery.fancybox.pack.js [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+if(27===c&&b.coming)return!1;!e.ctrlKey&&(!e.altKey&&!e.shiftKey&&!e.metaKey&&(!k||!k.type&&!f(k).is("[contenteditable]")))&&f.each(d,function(d,k){if(1<a.group.length&&k[c]!==v)return b[d](k[c]),e.preventDefault(),!1;if(-1<f.inArray(c,k))return b[d](),e.preventDefault(),!1})}),f.fn.mousewheel&&a.mouseWheel&&b.wrap.bind("mousewheel.fb",function(d,c,k,g){for(var h=f(d.target||null),j=!1;h.length&&!j&&!h.is(".fancybox-skin")&&!h.is(".fancybox-wrap");)j=h[0]&&!(h[0].style.overflow&&"hidden"===h[0].style.overflow)&&
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:16]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: _site/js/jquery.fancybox.pack.js [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+(h[0].clientWidth&&h[0].scrollWidth>h[0].clientWidth||h[0].clientHeight&&h[0].scrollHeight>h[0].clientHeight),h=f(h).parent();if(0!==c&&!j&&1<b.group.length&&!a.canShrink){if(0<g||0<k)b.prev(0<g?"down":"left");else if(0>g||0>k)b.next(0>g?"up":"right");d.preventDefault()}}))},trigger:function(a,d){var e,c=d||b.coming||b.current;if(c){f.isFunction(c[a])&&(e=c[a].apply(c,Array.prototype.slice.call(arguments,1)));if(!1===e)return!1;c.helpers&&f.each(c.helpers,function(d,e){if(e&&b.helpers[d]&&f.isFunction(b.helpers[d][a]))b.helpers[d][a](f.extend(!0,
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:17]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.
-  Confusing use of '!'.

File: _site/js/jquery.fancybox.pack.js [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+{},b.helpers[d].defaults,e),c)});p.trigger(a)}},isImage:function(a){return q(a)&&a.match(/(^data:image\/.*,)|(\.(jp(e|g|eg)|gif|png|bmp|webp|svg)((\?|#).*)?$)/i)},isSWF:function(a){return q(a)&&a.match(/\.(swf)((\?|#).*)?$/i)},_start:function(a){var d={},e,c;a=l(a);e=b.group[a]||null;if(!e)return!1;d=f.extend(!0,{},b.opts,e);e=d.margin;c=d.padding;"number"===f.type(e)&&(d.margin=[e,e,e,e]);"number"===f.type(c)&&(d.padding=[c,c,c,c]);d.modal&&f.extend(!0,d,{closeBtn:!1,closeClick:!1,nextClick:!1,arrows:!1,
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:18]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: _site/js/jquery.fancybox.pack.js [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+mouseWheel:!1,keys:null,helpers:{overlay:{closeClick:!1}}});d.autoSize&&(d.autoWidth=d.autoHeight=!0);"auto"===d.width&&(d.autoWidth=!0);"auto"===d.height&&(d.autoHeight=!0);d.group=b.group;d.index=a;b.coming=d;if(!1===b.trigger("beforeLoad"))b.coming=null;else{c=d.type;e=d.href;if(!c)return b.coming=null,b.current&&b.router&&"jumpto"!==b.router?(b.current.index=a,b[b.router](b.direction)):!1;b.isActive=!0;if("image"===c||"swf"===c)d.autoHeight=d.autoWidth=!1,d.scrolling="visible";"image"===c&&(d.aspectRatio=
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:19]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.
-  Confusing use of '!'.

File: _site/js/jquery.fancybox.pack.js [hunk 15]
```diff
@@ -0,0 +1,46 @@
...
+!0);"iframe"===c&&s&&(d.scrolling="scroll");d.wrap=f(d.tpl.wrap).addClass("fancybox-"+(s?"mobile":"desktop")+" fancybox-type-"+c+" fancybox-tmp "+d.wrapCSS).appendTo(d.parent||"body");f.extend(d,{skin:f(".fancybox-skin",d.wrap),outer:f(".fancybox-outer",d.wrap),inner:f(".fancybox-inner",d.wrap)});f.each(["Top","Right","Bottom","Left"],function(a,b){d.skin.css("padding"+b,w(d.padding[a]))});b.trigger("onReady");if("inline"===c||"html"===c){if(!d.content||!d.content.length)return b._error("content")}else if(!e)return b._error("href");
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:20]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: _site/js/jquery.fancybox.pack.js [hunk 16]
```diff
@@ -0,0 +1,46 @@
...
+"image"===c?b._loadImage():"ajax"===c?b._loadAjax():"iframe"===c?b._loadIframe():b._afterLoad()}},_error:function(a){f.extend(b.coming,{type:"html",autoWidth:!0,autoHeight:!0,minWidth:0,minHeight:0,scrolling:"no",hasError:a,content:b.coming.tpl.error});b._afterLoad()},_loadImage:function(){var a=b.imgPreload=new Image;a.onload=function(){this.onload=this.onerror=null;b.coming.width=this.width/b.opts.pixelRatio;b.coming.height=this.height/b.opts.pixelRatio;b._afterLoad()};a.onerror=function(){this.onload=
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:21]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.
-  Missing '()' invoking a constructor.

File: _site/js/jquery.fancybox.pack.js [hunk 17]
```diff
@@ -0,0 +1,46 @@
...
+this.onerror=null;b._error("image")};a.src=b.coming.href;!0!==a.complete&&b.showLoading()},_loadAjax:function(){var a=b.coming;b.showLoading();b.ajaxLoad=f.ajax(f.extend({},a.ajax,{url:a.href,error:function(a,e){b.coming&&"abort"!==e?b._error("ajax",a):b.hideLoading()},success:function(d,e){"success"===e&&(a.content=d,b._afterLoad())}}))},_loadIframe:function(){var a=b.coming,d=f(a.tpl.iframe.replace(/\{rnd\}/g,(new Date).getTime())).attr("scrolling",s?"auto":a.iframe.scrolling).attr("src",a.href);
```
Comment (subho007) [inline _site/js/jquery.fancybox.pack.js:22]: **[jshint]**
-  Confusing use of '!'.
-  Expected an assignment or function call and instead saw an expression.
-  Missing semicolon.
-  Expected an identifier and instead saw '==='.
-  Too many errors. (46% scanned).

File: _site/js/main.js [hunk 18]
```diff
@@ -0,0 +1,2551 @@
+/////////////////////////////////////////////////////////////////
+//Globally declaring $ as reference to jQuery as required in WP
+var $ = jQuery
```
Comment (subho007) [inline _site/js/main.js:3]: **[jshint]**
-  Redefinition of '$'.

File: _site/js/main.js [hunk 19]
```diff
@@ -0,0 +1,2551 @@
...
+}
+
+//Create the jump navigation for the slide
+AkSlideManager.prototype.createJumpers = function(){
+  var _this = this;
+  var html = "";
+
+  for(var i = 0; i < this.noOfSlides; i++){
+    var first = "";
+    if(i == 0){first = "ak-jump-active"}
```
Comment (subho007) [inline _site/js/main.js:154]: **[jshint]**
-  Use '===' to compare with '0'.

File: _site/js/main.js [hunk 20]
```diff
@@ -0,0 +1,2551 @@
...
+  clearInterval(_this.interval);
+  _this.interval = null;
+  _this.hideSlide(slideNumToBring);
+}
+
+AkSlideManager.prototype.onSwipeRight = function(ev){
+  var _this = this;
+  var slideNumToBring = 0;
+
+  if(_this.currentSlide == 0){
```
Comment (subho007) [inline _site/js/main.js:221]: **[jshint]**
-  Use '===' to compare with '0'.

File: _site/js/main.js [hunk 21]
```diff
@@ -0,0 +1,2551 @@
...
+  var elesToAnimate = $(currentSlideBlock).find("." + this.eleAnimateClass);
+  var timeToHide = 0;
+
+  for(var i=0; i < elesToAnimate.length; i++){
+    var animator = new AkElementAnimator(elesToAnimate[i]);
+    animator.perishAnimate();
+    timeToHide = animator.animTime > timeToHide ? animator.animTime : timeToHide;
+  }
+
+  if(nextSlideNum != undefined ){
```
Comment (subho007) [inline _site/js/main.js:262]: **[jshint]**
-  Use '!==' to compare with 'undefined'.

File: _site/js/main.js [hunk 22]
```diff
@@ -0,0 +1,2551 @@
...
+  _this.slideZone.prepend(appendMarginDefendLi);
+}
+
+AkSlideManager.prototype.adjustSlideContainerHeight = function(){
+  var clonedSlideZone = _this.slideZone.clone();
+  var newRandomId = "clonedSlideZone" + Math.floor(Math.random()*1000);
+}
+
+AkSlideManager.prototype.swipeHandler = function(){
+  if(this.slideZone == undefined || this.slideZone.length ==0){
```
Comment (subho007) [inline _site/js/main.js:305]: **[jshint]**
-  Use '===' to compare with 'undefined'.
-  Use '===' to compare with '0'.

File: _site/js/main.js [hunk 23]
```diff
@@ -0,0 +1,2551 @@
...
+  startY,
+  distX,
+  distY,
+  threshold = 100, //required min distance traveled to be considered swipe
+  restraint = 100, // maximum distance allowed at the same time in perpendicular direction
+  allowedTime = 200, // maximum time allowed to travel that distance
+  elapsedTime,
+  startTime;
+
+  var touchMoveEv = undefined;
```
Comment (subho007) [inline _site/js/main.js:321]: **[jshint]**
-  It's not necessary to initialize 'touchMoveEv' to 'undefined'.

File: _site/js/main.js [hunk 24]
```diff
@@ -0,0 +1,2551 @@
...
+}
+
+AkElementAnimator.prototype.createAnimator = function(element){
+  this.initialCss = JSON.parse($(element).attr("data-init-css"));
+  this.appearanceCss = JSON.parse($(element).attr("data-appear-css"));
+  this.perishCss = JSON.parse($(element).attr("data-perish-css"));
+  this.animTime = parseInt($(element).attr("data-ak-anim-time"));
+  this.animDelay = parseInt($(element).attr("data-ak-anim-delay"));
+
+  if(this.animTime === NaN){
```
Comment (subho007) [inline _site/js/main.js:389]: **[jshint]**
-  Use the isNaN function to compare with NaN.

File: _site/js/main.js [hunk 25]
```diff
@@ -0,0 +1,2551 @@
...
+    this.animTime = this.defaultAnimTime;
+  }
+
+  if(this.animDelay === NaN){
```
Comment (subho007) [inline _site/js/main.js:393]: **[jshint]**
-  Use the isNaN function to compare with NaN.

File: _site/js/main.js [hunk 26]
```diff
@@ -0,0 +1,2551 @@
...
+AkElementAnimator.prototype.setInitialState = function(){
+  var _this = this;
+  $(_this.eleToAnimate).stop().css(_this.initialCss);
+}
+
+AkElementAnimator.prototype.appearanceAnimate = function(){
+  var _this = this;
+  var animTime = _this.animTime;
+
+  if(_this.animDelay != 0){
```
Comment (subho007) [inline _site/js/main.js:407]: **[jshint]**
-  Use '!==' to compare with '0'.

File: _site/js/main.js [hunk 27]
```diff
@@ -0,0 +1,2551 @@
...
+  }else if(companyPage.length === 0 && doc.width() > 760){
+    $("#company-dropdown").removeClass("block");
+  }
+}
+
+function hideCompanySubMenu(ev){
+  var target = ev.currentTarget;
+  var companyPage = $("#company-page");
+
+  if(target.id !== "company-menu" && companyPage.length == 0){
```
Comment (subho007) [inline _site/js/main.js:528]: **[jshint]**
-  Use '===' to compare with '0'.

File: _site/js/main.js [hunk 28]
```diff
@@ -0,0 +1,2551 @@
...
+  }
+
+  try{
+    var mouseY = ev.clientY;
+    var subMenuTop = $("#company-dropdown").css("top");
+    var subMenuHeight = $("#company-dropdown").outerHeight();
+    var downEnd = parseInt(subMenuTop) + subMenuHeight;
+    var companyPage = $("#company-page");
+
+    if(companyPage.length == 0 && mouseY >= (downEnd + 100)){
```
Comment (subho007) [inline _site/js/main.js:548]: **[jshint]**
-  Use '===' to compare with '0'.

File: _site/js/main.js [hunk 29]
```diff
@@ -0,0 +1,2551 @@
...
+  }catch(e){}
+}
+
+////////////////////////////////////////////////////////
+//Method to prevent navigation of company-link in mobile menu
+function companyLinkClickAction(){
+  $("#company-link").click(function(ev){
+    var doc = $(document);
+    if(doc.width()<760){
+      ev.preventDefault;
```
Comment (subho007) [inline _site/js/main.js:560]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: _site/js/main.js [hunk 30]
```diff
@@ -0,0 +1,2551 @@
...
+  }else if(resourcePage.length === 0 && doc.width() > 760){
+    $("#resource-dropdown").removeClass("block");
+  }
+}
+
+function hideResourceSubMenu(ev){
+  var target = ev.currentTarget;
+  var resourcePage = $("#resource-page");
+
+  if(target.id !== "resource-menu" && resourcePage.length == 0){
```
Comment (subho007) [inline _site/js/main.js:610]: **[jshint]**
-  Use '===' to compare with '0'.

File: _site/js/main.js [hunk 31]
```diff
@@ -0,0 +1,2551 @@
...
+  }
+
+  try{
+    var mouseY = ev.clientY;
+    var subMenuTop = $("#resource-dropdown").css("top");
+    var subMenuHeight = $("#resource-dropdown").outerHeight();
+    var downEnd = parseInt(subMenuTop) + subMenuHeight;
+    var companyPage = $("#resource-page");
+
+    if(companyPage.length == 0 && mouseY >= (downEnd + 100)){
```
Comment (subho007) [inline _site/js/main.js:630]: **[jshint]**
-  Use '===' to compare with '0'.

File: _site/js/main.js [hunk 32]
```diff
@@ -0,0 +1,2551 @@
...
+  }catch(e){console.log(e.message)}
+}
+
+////////////////////////////////////////////////////////
+//Method to prevent navigation of resource-link in mobile menu
+function resourceLinkClickAction(){
+  $("#resource-link").click(function(ev){
+    var doc = $(document);
+    if(doc.width()<760){
+      ev.preventDefault;
```
Comment (subho007) [inline _site/js/main.js:642]: **[jshint]**
-  Expected an assignment or function call and instead saw an expression.

File: _site/js/main.js [hunk 33]
```diff
@@ -0,0 +1,2551 @@
...
+  this.deltaToShow = 60;
+}
+
+ResourceManager.prototype.init = function(){
+  var _this = this;
+
+  $(document).ready(function(){
+    var resourceZone =  $("#resources-zone");
+
+    if(resourceZone.length == 0){
```
Comment (subho007) [inline _site/js/main.js:723]: **[jshint]**
-  Use '===' to compare with '0'.

File: _site/js/main.js [hunk 34]
```diff
@@ -0,0 +1,2551 @@
...
+
+    _this.setResourceType();
+    _this.showResourceType(true);
+    _this.bindSubMenuClick();
+    _this.bindScroll();
+  });
+}
+
+ResourceManager.prototype.setResourceType = function(){
+  this.resourceType = document.location.hash == "" ? "resource" : document.location.hash.substring(1).toLowerCase();
```
Comment (subho007) [inline _site/js/main.js:735]: **[jshint]**
-  Use '===' to compare with ''.

File: _site/js/main.js [hunk 35]
```diff
@@ -0,0 +1,2551 @@
...
+
+  this.annualFactor = $("#" + _this.annualFactorId).attr("data-attr-anf");
+  this.inputBox = $("#noOfAppsInput");
+  this.plusOne = $("#appPlus");
+  this.minusOne = $("#appMinus");
+  this.minAppCount = this.inputBox.attr("data-min");
+  this.maxAppCount = this.inputBox.attr("data-max");
+  this.basePrice = this.inputBox.attr("data-base-price");
+  this.annualFactor = this.inputBox.attr("data-annual-factor");
+  this.deltaPriceMinus = this.inputBox.attr("data-delta-minus");;
```
Comment (subho007) [inline _site/js/main.js:1081]: **[jshint]**
-  Unnecessary semicolon.

**Comment by subho007** [comment]
@yashwin please approve the changes

**Comment by yashwin** [comment]
@subho007 : Do I still need to do it? You merged it, right?

**Comment by subho007** [comment]
@yashwin nopes not neede 👍 
I'm changing this repo, removing the master branch. From now on, the master branch is the source branch

**Comment by yashwin** [comment]
@subho007 : Ok 👍

**Pull request merged**